universal-login-react/src/ui/styles/assetsItem.sass

```diff
@@ -25,7 +25,13 @@
     font-size: 14px
     text-transform: capitalize
 
-  .assets-price
+  .assets-price-left
```

Comment:
You already have a parent class indicating positon `assets-item-left`,

So you can simply do:
```ts
.assets-item-left
  .assets-price
    text-align: left
```

No need to introduce new classes.